packages/nextjs/src/tools/codegen/utils.ts

```diff
@@ -207,31 +209,31 @@ export const resolveComponentImportFiles = (
 export const sendCode = async ({
   file,
   token,
-  edgeUrl,
+  targetUrl,
 }: {
   file: ExtractedFile;
   token: string;
-  edgeUrl?: string;
+  targetUrl: string;
 }) => {
-  const meshEndpoint = `${edgeUrl || constants.SITECORE_EDGE_URL_DEFAULT}/api/v1/mesh`;
+  const apiEndpoint = `${targetUrl}/api/v1/contentsdk/code/extracted`;
```

Comment:
we are still using the mesh endpoint right?

Reply:
Mesh will be incorporated into edge proxy - but it isn't currently. We use MESH_URL as an option for now, so that we can test and not block other development.
Once mesh is incorporated, we will only use edgeUrl - which will be passed as targetUrl here.